### PR TITLE
target layer for mobile view

### LINKED ISSUE #1733: Modal should stay within mobile view in Wizard when canceling
Screen Shot 2021-07-14 at 2 16 22 PM
I think we should show an example of a much smaller modal that fits within the mobile view (see image). I've attached an example below of how it is in Figma.

https://www.figma.com/file/GEhgYzx3b8MejBjg9gAxyA/Wizarding?node-id=127%3A892

I'm also questioning whether we needs to shrink down the text within the button. It's rather long

### PR DESCRIPTION
**Pull request by britt6612**
[Deploy Preview](https://deploy-preview-2142--keen-mayer-a86c8b.netlify.app/)
#### What does this PR do?
Changes the target  of the `layer`
#### Where should the reviewer start?
wizard/components/cancellationLayer
#### What testing has been done on this PR?
mobile view
In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### How should this be manually tested?
mobile view for cancel in Layer
#### What are the relevant issues?
closes #1733 
#### Should this PR be mentioned in Design System updates?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

**Review by MikeKingdom — COMMENTED** [review]
The cancel modal on mobile (iPhone  X) is currently looking like this which is different than the Figma. 
image

**Comment by britt6612** [comment]
> The cancel modal on mobile (iPhone X) is currently looking like this which is different than the Figma. image

I will take a look at that one!

**Review by taysea — COMMENTED** [review]
I am seeing this For TwoColumnWizard on mobile:
Screen Shot 2021-12-14 at 9 27 18 AM

And this for Validation:
Screen Shot 2021-12-14 at 9 27 44 AM

I think in both instances we want it to stay within the bounds of the example and take the full height?

File: aries-site/src/examples/templates/wizard/TwoColumnWizardExample.js [hunk 1]
```diff
@@ -176,6 +176,7 @@ const steps = [
 ];
 
 export const TwoColumnWizardExample = () => {
```
[the remark this answers is not in the document]
Comment (britt6612) [reply aries-site/src/examples/templates/wizard/TwoColumnWizardExample.js]: thank you for mentioning we have `containerRef` used this instead

File: aries-site/src/examples/templates/wizard/components/CancellationLayer.js [hunk 2]
```diff
@@ -25,7 +28,7 @@ export const CancellationLayer = ({ onSetOpen }) => {
         <Box
           as="footer"
           gap="small"
-          direction="row"
+          direction={size !== 'small' ? 'row' : 'column'}
           align="center"
           justify="end"
```
Comment (taysea) [inline aries-site/src/examples/templates/wizard/components/CancellationLayer.js:33]: Based on recent guidance, on mobile the button in this box should take full-width -- can we add `fill="horizontal"` when size is small?

File: aries-site/src/examples/templates/wizard/TwoColumnWizardExample.js [hunk 3]
```diff
@@ -175,7 +176,7 @@ const steps = [
   },
 ];
 
-export const TwoColumnWizardExample = () => {
+export const TwoColumnWizardExample = ({ containerRef }) => {
```
Comment (taysea) [inline aries-site/src/examples/templates/wizard/TwoColumnWizardExample.js:179]: Can we copy and paste the comment from the Filtering file about why this prop is here?

Comment (britt6612) [reply aries-site/src/examples/templates/wizard/TwoColumnWizardExample.js:179]: yes I will add

File: aries-site/src/examples/templates/wizard/components/CancellationLayer.js (hunk 2 above)
Comment (britt6612) [reply aries-site/src/examples/templates/wizard/components/CancellationLayer.js:33]: yes I actually had to add it to `button` but looks better nice catch!

**Review by MikeKingdom — APPROVED** [review]
Thanks Brittany!

**Review by taysea — APPROVED** [review]
LGTM!

**Pull request merged**